syntax = "proto3";


option go_package = "./;pb";

service TextProcessingService{
    rpc ProcessText(ProcessingRequest) returns (ProcessingResponse);
}

message ProcessingRequest {
    string text = 1;
}
message ProcessingResponse {
    string text = 1;
}